CHANGELOG.md

```diff
@@ -27,10 +27,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.21.5] - 2023-05-08
 
-### Fixed
-- Updated container build script to pull in the correct kubectl version from the
-  metal-provision repository instead of the csm-rpms repository.
-
```

Comment:
I realize this came from a revert, but I wonder if we should just leave this entry alone and add a new 3.21.6 version that has an entry describing the change.